pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -2489,6 +2489,29 @@ buildLuarocksPackage {
   };
 }) {};
 
+lze = callPackage({ buildLuarocksPackage, fetchurl, fetchzip, luaOlder }:
+buildLuarocksPackage {
+  pname = "lze";
+  version = "1.5.0-1";
+  knownRockspec = (fetchurl {
+    url    = "mirror://luarocks/lze-1.5.0-1.rockspec";
+    sha256 = "1ag7kczm71s6wfjhsahcmxxj8xizxb2xlk6fcw5lvgqvibvcv5w1";
```

Comment:
issue: Use `hash`.

Tip: You can `nix run .#luarocks-nix -- nix lze` to get the generated output manually.

Reply:
I did run that comand. This is what it gave me. It used sha256

```nix
nix run nixpkgs#luarocks-nix -- nix lze
path is '/nix/store/nq1k7y8b353mdsjg99jr9ycrv1r5iixx-v1.5.0.zip'
{ buildLuarocksPackage, fetchurl, fetchzip, luaOlder }:
buildLuarocksPackage {
  pname = "lze";
  version = "1.5.0-1";
  knownRockspec = (fetchurl {
    url    = "mirror://luarocks/lze-1.5.0-1.rockspec";
    sha256 = "1ag7kczm71s6wfjhsahcmxxj8xizxb2xlk6fcw5lvgqvibvcv5w1";
  }).outPath;
  src = fetchzip {
    url    = "https://github.com/BirdeeHub/lze/archive/v1.5.0.zip";
    sha256 = "05zgwb0jw5lnsqdbw3qs5025lz3w8w0bca72rzra5y2249hk8a4y";
  };

  disabled = luaOlder "5.1";

  meta = {
    homepage = "https://github.com/BirdeeHub/lze";
    description = "🦥 fork of lz.n, a dead simple lazy-loading Lua library for Neovim plugins, focused on extensibility";
    license.fullName = "GPL-2+";
  };
}
```

Reply:
Thats what I put in the original commit because the generator kept failing on random things

Reply:
https://github.com/NixOS/nixpkgs/blob/018056326d2f8d1ccd31a58c4e650d9e1823a27e/pkgs/development/lua-modules/generated-packages.nix#L2469-L2490

In fact, yours also is the same.

Reply:
Strange, I thought the latest version of luarocks-nix output `hash`.
But maybe I'm mistaken. Anyways, as long as this is generated using luarocks-nix from this package set, it's not an issue.

Reply:
Also, thank you, by the way, for showing me more of the value in testing, and also for showing me some cool github tricks. Your packaging knowledge is seriously impressive.